projects/storefrontlib/src/cms-components/myaccount/order/amend-order/cancellations/cancel-order-confirmation/cancel-order-confirmation.component.html

```diff
@@ -18,7 +18,7 @@
     <cx-amend-order-actions
       *ngIf="orderCode"
       [orderCode]="orderCode"
-      [isValid]="form.valid"
+      [control]="form"
```

Comment:
Maybe rename control with form?

Reply:
You might be right here. Renamed to *form*.